wiki/608450-SynapseEnvironment.md

```diff
@@ -0,0 +1,96 @@
+## Overview
+
+Synapse is a collaborative platform for researchers and developers. Synapse is
+developed by Sage Bionetworks to offer services that support:
+
+- Organization of digital research assets (e.g., files, SQL tables, Docker
+  images)
+- Recording provenance to give researchers credits for their work
+- Collaboration among researchers
+
+The NLP Sandbox walk in the footsteps of previous successful scientific
+challenges that been organized using Synapse. The NLP Sandbox relies on the
+following features of Synapse to enable its users to develop and benchmark new
+NLP Tools.
+
+- Creation and management of Teams
+- Unlimited storage of private Docker images
+- Benchmark the performance of NLP Tools on public and private datasets
+
+The sections below describe how to setup your development environment on Synapse
+so you can start developing and benchmarking your next NLP tool.
+
+## Create a Synapse account
+
+If you don't have already an account, visit the home page of [synapse.org] and
+register. Make sure to complete your profile information.
+
+## Get Certified
+
+You need to certify your Synapse account if you want to perform the following
+actions:
+
+- Update files to Synapse
```

Comment:
```suggestion
- Update files in Synapse
```

Just to clarify, is this to update or to upload?

Reply:
@ychae Upload